yarn-project/circuits.js/src/structs/proof.ts

```diff
@@ -16,6 +17,8 @@ export class Proof {
      * Holds the serialized proof data in a binary buffer format.
      */
     public buffer: Buffer,
+
+    public numPublicInputs = 0,
```

Comment:
A default argument here scares me a little.

Reply:
Yeah, let's not default this. Should only need to be updated in a couple of places.